frontend/server/src/main/java/org/pytorch/serve/metrics/MetricCollector.java

```diff
@@ -32,9 +32,11 @@ public MetricCollector(ConfigManager configManager) {
     public void run() {
         try {
             // Collect System level Metrics
-            String[] args = new String[2];
+            String[] args = new String[4];
             args[0] = configManager.getPythonExecutable();
             args[1] = "ts/metrics/metric_collector.py";
+            args[2] = "--gpu";
```

Comment:
what will call this code? How do we GPU metrics aren't instrumented for CPU only runs

Reply:
The action "etric_collector.py" is scheduled by Metrics Manager. Parameter "--gpu" is used to tell backend the host type (ie. CPU or GPU).